components/License/helper.ts

```diff
@@ -19,33 +19,42 @@ const options: string[] = [
   'marketingEndorsement',
 ];
 
-const optionValue = options.reduce((optionValue: OptionValue, option) => {
-  optionValue[option] = [
-    { value: FeatureAttitude.Undefined, text: t('feature_attitude_undefined') },
-    {
-      value: FeatureAttitude.Positive,
-      text: t('feature_attitude_positive'),
+const optionValue: () => OptionValue = () => {
+  const optionValue: OptionValue = options.reduce(
+    (optionValue: OptionValue, option) => {
```

Comment:
```suggestion
const optionValue = () => {
  const optionValue = options.reduce(
    (optionValue, option) => {
```
数据起始位置标清类型，后面都能推导（见下一条）